include/geom/cell_pyramid.h

```diff
@@ -151,6 +151,8 @@ class Pyramid : public Cell
 
   virtual std::vector<unsigned int> sides_on_edge(const unsigned int e) const override final;
 
+  bool contains_point (const Point & p, Real tol=TOLERANCE) const override;
```

Comment:
No final here?

Reply:
Hmm - I'm actually not accustomed to using final at all. Didn't even think about it. Will add.

Reply:
Doesn't actually make a difference in this case, just a good habit to have.  In other cases it can make the difference between "this will be a virtual function call" versus "this is inlined".